packages/cli/src/Version.ts

```diff
@@ -91,7 +92,10 @@ export class Version implements Command {
     const rows = [
       [packageJson.name, packageJson.version],
       ['@prisma/client', prismaClientVersion ?? 'Not found'],
-      ['Current platform', platform],
+      ['Current platform', os.platform()],
```

Comment:
Changing the meaning of this is a bit "meh", but I think I am ok with it.

Maybe "Current OS" instead to avoid the repurposing?

(And then maybe start with "Computed binaryTarget", then OS and architecture)

Reply:
Sounds good.

I changed the wording to 
```
      ['Computed binaryTarget', platform],
      ['Operating System', os.platform()],
      ['Architecture', os.arch()],
```

Reply:
`Current` was not needed